inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/workload/DynamicWorkloadManagementPage.java

```diff
@@ -94,14 +92,12 @@
 
     private Form<Void> searchForm;
 
-
     // SpringBeans
     private @SpringBean UserDao userRepository;
     private @SpringBean ProjectService projectService;
     private @SpringBean MenuItemRegistry menuItemService;
     private @SpringBean DocumentService documentService;
-    private @SpringBean WorkloadProperties workloadProperties;
-    private @SpringBean DefaultAnnotationsProperties defaultAnnotations;
+    private @SpringBean WorkloadAndWorkflowService workloadAndWorkflowService;
 
     //Default constructor, no project selected (only when workload.html
     // put directly in the browser without any parameters)
```

Comment:
We use line comments **in** methods. We use JavaDoc to describe what methods do - if what they do is not obvious.